comms/server/server.go

```diff
@@ -148,13 +150,20 @@ func getMessages(c echo.Context) error {
 		return c.String(400, "wallet not found: "+err.Error())
 	}
 
-	params := queries.ChatMessagesAndReactionsParams{UserID: int32(userId), ChatID: c.Param("id"), Cursor: time.Now().UTC(), Limit: 50}
-	if c.QueryParam("cursor") != "" {
-		cursor, err := time.Parse(time.RFC3339Nano, c.QueryParam("cursor"))
+	params := queries.ChatMessagesAndReactionsParams{UserID: int32(userId), ChatID: c.Param("id"), Before: time.Now().UTC(), After: time.Time{}, Limit: 50}
```

Comment:
should `After` also be `time.Now().UTC()`

Reply:
I don't think so - I think it should default to beginning of time to include as much as possible?